### DISCUSSION TITLE
Mastodon v4.2.0  Posting box & dropdown menu   General - bug

### DISCUSSION THREAD

**Discussion by Schawari** [mastodon/mastodon#27323] [General]
Hello to the developement of Mastodon and to everybody :)

After the update to v4.2.0 I have some usage problems.
I have screens from mullvad + firefox browser.
I have now sorted the problems under bugs + hope that it is correct.
Excuse my not perfect english^^ (Have used deepl)
Thx for your attention :)

1. When I reply to a post, the **posting box** is now very narrow that I can just see 2.5 lines of what I wrote. 
This was previously only the case when replying to 1 CW post.
My browser view is zoomed to 120%, but the mentioned problem also occurs at 100% (laptop 1920x1080).

2. When I click the **dropdown menu** on the left (the 3 dashes), the last two items are no longer visible in the 120% view. Has the dropdown menu become longer?

**Mastodon instance**
fnordon.de  just USER!

**Mastodon version**
4.2.x 
https://github.com/mastodon/mastodon

**Browser name and version**
Mullvad 12.5.6 (based on Firefox 102.15) (64 bit)
Firefox 118.0.1  (64 Bit)

**Operating system**
Windows 10

**Mullvad**:

![Bug1Mullvad](https://github.com/mastodon/mastodon/assets/147314134/c953a71e-9bc8-400f-be08-d2019ddbf347)
![Bug2Mullvad](https://github.com/mastodon/mastodon/assets/147314134/ce1e3047-9b57-473b-9c9b-ea35f7cfac83)

**Firefox**

![Bug1Firefox](https://github.com/mastodon/mastodon/assets/147314134/a4db980e-1d0d-41f1-9290-bb7a66bafb4d)
![Bug2Firefox](https://github.com/mastodon/mastodon/assets/147314134/cf534c3b-68de-4e1a-8c0d-976b688bddab)

**Comment by mjankowski** [comment]
Do you still have this issue on latest 4.3 release?
